config/routes.rb

```diff
@@ -22,6 +22,8 @@
   # route for chatfuel questions and answers
   get 'chatfuel/text_components/:text_component_id/answer_to_question/:index', to: 'chatfuel#answer_to_question', as: 'answer_to_question'
 
+  get '/smaxtec_api/temperature', to: 'smaxtec_api#get_temperature'
```

Comment:
and https://github.com/roschaefer/story.board/blob/master/lib/tasks/archive.rake